ui/src/app/components/resource-card.spec.tsx

```diff
@@ -122,21 +122,21 @@ describe('ResourceCardComponent', () => {
   // without a browser to define bounding boxes...
   it('should always have a clickable popup menu (cohort)', () => {
     const wrapper = component(ResourceCardWrapper.readonlyCohortCard);
-    expect(wrapper.find('[data-test-id="resource-menu"]').props()['disabled']).toBeUndefined();
+    expect(wrapper.find('[data-test-id="resource-menu"]').props()).not.toContain('disabled');
```

Comment:
🏅 Much better. 

One other thing that might make it more readable is to pull out a helper method to fetch the resource menu (or its props), since you do that in several places.